src/main/kotlin/androidx/compose/material/icons/generator/IconWriter.kt

```diff
@@ -60,7 +61,8 @@ class IconWriter(
             val (fileSpec, accessProperty) = VectorAssetGenerator(
                 iconName,
                 groupPackage,
-                vector
+                vector,
+                generatePreview
```

Comment:
Missed `generatePreview` in https://github.com/DevSrSouza/svg-to-compose/blob/8a145f5b0ff7810a3abb644e932778f9beb8bc9a/src/main/kotlin/br/com/devsrsouza/svg2compose/Svg2Compose.kt#L87-L91

Reply:
Fixing in #22.

Reply:
Just noticed it was also addressed to #20.